lib/config/oldDefault.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
```

Comment:
do we need this file any longer?

Reply:
We probably don't need it. I kept it to either migrate and also keep it consistent.